nx/test/nx/lin_alg_test.exs

```diff
@@ -426,14 +427,50 @@ defmodule Nx.LinAlgTest do
                ])
     end
 
+    test "correctly a eigenvalues and eigenvectors for a Hermitian matrix case" do
+      # Hermitian matrix
+      t =
+        Nx.tensor([
+          [1, Complex.new(0, 2), 2],
+          [Complex.new(0, -2), -3, Complex.new(0, 2)],
+          [2, Complex.new(0, -2), 1]
+        ])
+
+      assert {eigenvals, eigenvecs} = Nx.LinAlg.eigh(t)
+
+      # Eigenvalues
+      assert eigenvals ==
+               Nx.tensor([Complex.new(-5, 0), Complex.new(3, 0), Complex.new(1, 0)])
+
+      # Eigenvectors
+      assert complex_round(eigenvecs, 3) ==
+               Nx.tensor([
```

Comment:
shouldn't these results also be rounded to 3 places?

Reply:
Does it mean that the value should not be rounded ?

Reply:
I meant that it doesn't make sense to write out that many decimal places if you're comparing to a rounded-to-3-places vector on the left